docs/getting-started.md

```diff
@@ -31,9 +31,25 @@ There's also an extra package: `@ecl-twig/{ec,eu}-components` which is used to a
 
 Each component comes with a story `*.story.js` which is consumed by [Storybook](https://storybook.js.org/). In this story file, you import the main Twig template and you pass data to it. That's it!
 
-You can use knobs to make the showcase more dynamic and interactive.
+In ecl-twig the storybook knobs are extensively used to accurately represent the full input data with the aim of offering the possibility to customise the component and retrieve the correct object to the passed to the component in the `notes` tab.
+Their label is supposed to be defined the same way the property is accessible in the expected input data.
+Ex: `link.icon.path`
 
-More features will come in the near future...
+Knobs in ecl-twig are grouped together to simplify their usage and offer additional information, like whether the property controlled by the knob is mandatory or optional, for instance, the available groups are _mandatory_, _optional_, _use cases_, _status_
+In ecl-twig buttons knobs are used to defined different use cases the component might have, when blocks of information are optional and can be toggled on and off in a component demo.
+In many cases (like icons, path to the logo...) there is no real advantage for the user in editing the value used in the demo, therefore we only offer the possibility of unsetting it, mainly for testing the compliance checks we perform.
+
+### ECL Compliance
+
+The ecl-twig library comes with a special component named `ecl-compliance`, which defines a parent template for the ecl compliance checks to be performed, it also contains all the children templates for the checks applied to the single components.
+Each of our templates should have a corresponding child template defined so to make a compliance check available through our storybook demo and to all the consumers of the library thanks to the ecl-compliance package and a parameter applied to the component instantiation: `_ecl_compliance_`.
+
+An additional parameter `_ecl_compliance_inner_check_` is available for identifying the inner checks run by a component on all its included templates from the overall compliance performed on a component. This offers us the possibility of differentiating the output of the report. Mind that for this parameter to work also the `icon_path` needs to be available in order to render an icon for the success or the failure of the checks.
```

Comment:
1) low-value UI gimmicky (icon) imposes a requirement/dependency
2) popular flag is `verbose` and not `inner`. Yes, it's wording, but rarely anyone cares about inner work until module the software works. "Verbose" suggests that the user is debugging a reason something is not working and is ready to see the lower-level logic.

The sentence is of course too long and can be broken down.

Reply:
1) it's optional, if no `icon_path` is available it renders the standard output. `_compliance_inner_check_` It's also an internal parameter for the most, we know when we have an icon path available and when not, basically, the docs here are for the developer awareness of when it's possible to apply an "inner" report in the compliance checks added to a component he might be working on.

2) in this case verbose would just mean the opposite, here the idea is to make it more compact, instead, when we know it's an "automatic" check performed by the internal logic of the templates, the final users can simply ignore it, not much to do with it.